### ISSUE TITLE
Subtitle (closed caption) support

### ISSUE THREAD

**Issue by louanbastos** [alx-xlx/goindex#13]
Please, if possible, add Closed Caption (Subtitle) support. Example: A video called example.mp4 and have a file called example.vtt - the player can recognize the subtitle name as the same of the video, so, always when a file directory has a video and a .vtt file, will show the subtitle, if don't have it, won't show nothing.

**Comment by vomeraine** [comment]
No, i think it would be better to do this option as VLC or PLEX doing. To make it clear, they are recognizing the caption like this: [Video_Name].[Language_Code].[File Extention]
example: 
Avengers.mp4 video 
Avengers.eng.srt English,
Avengers.fre.srt French...
AFAIK all other video players (nplayer, mpc-hc) are using external subs this way too,
This country code pages could be useful 
[ISO-639-1 codes](http://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) (2-letter)
[ISO-639-2/B codes](http://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) (3-letter).

And could you add a load custom subtitle section like [this](https://janacloud.ga/0:video/LzA6L01vdmllcy9Ib2xseXdvb2QlMjBNb3ZpZXMvSnVzdGljZSUyMExlYWd1ZSUyMFNueWRlcnMlMjBDdXQlMjAoMjAyMSklMjBFbmdsaXNoJTIwKDEwODBwJTIwSEJPLU1heCUyMFdlYlJpcCUyMEFBQzIuMCUyMHgyNjQpJTIwLSUyMFtTaGlOb2JpXS9KdXN0aWNlLkxlYWd1ZS5TbnlkZXJzLkN1dC4yMDIxLkVuZ2xpc2guMTA4MHAuSEJPLU1BWC5XRUJSaXAuQUFDMi4wLngyNjQgLSBbU2hpTm9iaV0ubWt2) man did. It would be better to add subtitle from pc.
Thank you ^_^!!!!
